src/__tests__/reducers/tasksReducer.test.js

```diff
@@ -0,0 +1,60 @@
+import tasksReducer from '../../redux/reducers/tasksReducer'
+import { SET_SETTINGS, SET_ANSWERS, } from '../../constants'
+
+describe('tasksReducer', () => {
+  test('should return a correct state after receiving a nonexistent action', () => {
+    const action = { type: 'NON_EXISTENT_ACTION', }
+    const initialState = {
```

Comment:
Better import this from reducer file

Reply:
This is to I can understand with which action the test has crashed

Reply:
I has used fakeData. when will be API the initial state of tags and words reducers will have changed